### ISSUE TITLE
For each Level decrease the time!!

### ISSUE THREAD

**Issue by patcami91** [nikocall/GithubAssignment#4]
You need to increase the difficulty for the player by giving him less time !!!

**Comment by nikocall** [comment]
yes the time will be reduced, also the assets(obstacles) will add up to make the levels harder.
